src/display/canvas.js

```diff
@@ -553,7 +553,7 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
         ctx.putImageData(chunkImgData, 0, i * FULL_CHUNK_HEIGHT);
       }
     } else {
-      error('bad image kind: ' + imgData.kind);
+      throw new Error(`bad image kind: ${imgData.kind}`);
```

Comment:
It comes from already parsed data -- FormatError usually thrown as result of PDF parsing.